CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ New Features
         - ``VacuumGuidingCenterTrajectory`` : Integrates the particle motion by vacuum guiding center ODEs, conserving energy and mu.
         - ``SlowingDownGuidingCenterTrajectory`` : Integrates the particle motion by vacuum guiding center ODEs with slowing down on electrons and main ions.
     - Particle trajectories can be plotted with ``desc.plotting.plot_particle_trajectories`` function.
+    - ``trace_particles`` function is differentiable. Both modes of the automatic differentiation, forward or reverse, are possible by proper choice of ``adjoint`` argument of the function. See ``diffrax`` documentation for details.
```

Comment:
have you checked what forward and reverse mode do if the particle leaves the domain? Do they give NaN for the whole trajectory then?